corehq/apps/locations/queryutil.py

```diff
@@ -20,22 +20,29 @@ class ComparedQuerySet(object):
     """Compare results and times of MPTT and CTE queries"""
 
     def __init__(self, mptt_set, cte_set, timing_context):
-        self._mptt_set = mptt_set
+        self._mptt_set = None if settings.IS_LOCATION_CTE_ONLY else mptt_set
```

Comment:
if `settings.IS_LOCATION_CTE_ONLY`, then `mptt_set` should always already be `None`, right?  If not, then I'd think there was a bug elsewhere.  You could replace this with
```python
if settings.IS_LOCATION_CTE_ONLY:
    assert mptt_set is None
```

Reply:
offlined. going to do this in a separate PR.